.github/workflows/integration-tests.yml

```
@@ -29,7 +29,18 @@ jobs:
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
 
-      - name: Integration tests with pytest
+      - name: Integration tests with pytest, holesky fork
+        run: |
+          poetry run pytest tests -m integration
+        env:
+          WEB3_RPC_ENDPOINTS: ${{ secrets.WEB3_RPC_ENDPOINT }}
```

you should use different name here for holesky
Maybe
```suggestion
          WEB3_RPC_ENDPOINTS: ${{ secrets.HOLSKY_WEB3_RPC_ENDPOINT }}
```

---

Done